all_in_one_seo_pack.php

```diff
@@ -302,7 +302,7 @@ function aiosp_plugin_row_meta( $actions, $plugin_file ) {
 
 			);
 
-		return aiosp_action_links( $actions, $plugin_file, $action_links, 'after' );
+			return aiosp_action_links( $actions, $plugin_file, $action_links, 'after' );
```

Comment:
Doesn't look like there was any edit here, and appears to have 1 too many indents. This change here could probably be discarded.

Reply:
I left this as is just for now.